Makefile

```diff
@@ -108,7 +108,7 @@ cleanall: clean cleanall-deps
 DOCKER_CMD ?= "docker"
 
 build-base: install-deps protoc build-server build-gateway build-beat build-agent build-bootstrap build-monit
-build: build-base buildall-cli
+build: build-base build-cli
```

Comment:
This isn't a good idea right now -- and anyway this PR is not the right place to change how we use the rules -- but you can optimize the CI build by calling the specific rules you want so time isn't being wasted cross-compiling for all OS target.s